### ISSUE TITLE
TWLoader won't work on Emunand

### ISSUE THREAD

**Issue by Zakkious** [Robz8/TWLoader#459]
### Your 3DS

Tell us which console do you have:

- [x]  Old 3DS
- [ ]  Old 3DS XL
- [ ]  2DS
- [ ]  New 3DS
- [ ]  New 3DS XL
- [ ]  New 2DS XL

### CFW and configuration

Tell us which CFW do you have:

- [x]  Boot9Strap (SigHax)
- [ ]  A9LH (Only Sysnand)
- [ ]  A9LH + Emunand
- [x]  Luma3DS
- [ ]  AuReiNAND
- [ ]  Other CFW [specify]
- [ ]  Older CFW
- [ ]  Not using CFW / Using TWLoader in 3DSX format

### Console version

Tell us which version do you have:

- [x]  >= 11.4
- [ ]  < 11.4

### TWLoader version

Tell us which versions you are using:

- [x]  Latest release
- [ ]  I've compiled myself _[insert here commit hash]_
- [ ]  Old release

### Expected behaviour

TWLoader should load the games 

### Actual behaviour

TWLoader don't load any game on Emunand,only on Sysnand.

### Steps to reproduce

1.Start TWLoader
2.Try to load game
3.Infinite Whitescreen

NOTE:  It does work,but only on SDCards without an Emunand installed.

**Comment by Robz8** [comment]
Did you install the TWLNAND sides on SysNAND?

**Comment by Zakkious** [comment]
Yep. The weird part is that TWLoader only work for me if it's in a sdcard without emunand.Otherwise ,it won't work

**Comment by KuraiKokoro** [comment]
Using EmuNand on o3dsxl, b9s, luma, latest firmware and version, cannot reproduce
